### PR TITLE
Admin layer list

### PR DESCRIPTION
**Pull request by LindaAlblas**

### CODE REVIEW

File: client/src/app/admin/layers/layer-list.component.ts [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+   * Gets all the users
+   */
+  getLayers() {
+     this.errorGettingLayerList = false;
+     this.layerService.getLayers()
+            .subscribe(
+            data => {
+              // Success
+              // TODO: let the API do this?
+              this.layerList = [];
```
Comment (alastaircarson) [inline client/src/app/admin/layers/layer-list.component.ts]: move aray initialisation above? Duplicated for error section

File: client/src/app/admin/layers/layer.component.ts [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+      this.getLayer();
+  }
+
+  /**
+   * Save layer details
+   * TODO
+   */
+  save(){
+      this.editMode = false;
+      this.getLayer();
```
Comment (alastaircarson) [inline client/src/app/admin/layers/layer.component.ts]: Not wired up yet?

File: server/api/layer_api.py [hunk 3]
```diff
@@ -57,3 +57,51 @@ def get(self, map_category):
...
             current_app.logger.critical(error_msg)
             return {'error': error_msg}, 500
+
+
+class LayerAPI(Resource):
+
+    @token_auth.login_required
+    def get(self, id):
+        """
+        Gets layers grouped by Category
```
Comment (alastaircarson) [inline server/api/layer_api.py]: Incorrect comment

File: client/src/app/admin/layers/layer.component.ts (hunk 2 above)
Comment (LindaAlblas) [inline client/src/app/admin/layers/layer.component.ts]: No, this is not wired up yet. The next story is about updating layer details.

**Comment by LindaAlblas** [comment]
@alastaircarson I have changed the layer API docs and moved initialising the layer list

**Pull request merged**